lib/seahorn/HornCex.cc

```diff
@@ -302,7 +303,7 @@ bool HornCex::runOnFunction(Module &M, Function &F) {
   LOG("cex", errs() << "Validated CEX by BMC engine.\n";);
 
   // get bmc trace
-  BmcTrace trace(bmc.getTrace());
+  BmcTrace<BmcEngine, ZModel_ref> trace(bmc.getTrace<BmcEngine, ZModel_ref>());
```

Comment:
This looks wrong. `getTrace` should not need to have template parameters specified. It gets them from its class `bmc`